.github/workflows/tpip-check.yml

```diff
@@ -31,20 +33,48 @@ jobs:
 
       - name: Generate TPIP Report
         run:  |
-          go-licenses report . --ignore github.com/Open-CMSIS-Pack/cbuild --template ../../scripts/template/tpip-license.template > ../../scripts/tpip.md
-          date +"%Y/%m/%d %T" >> ../../scripts/tpip.md
+          go-licenses report . --ignore github.com/Open-CMSIS-Pack/cbuild --template ../../scripts/template/tpip-license.template > ../../scripts/${{ env.report_name }}
+          date +"%Y/%m/%d %T" >> ../../scripts/${{ env.report_name }}
         working-directory: ./cmd/cbuild
         
-      - name: Save Changes
+      - name: Archive TPIP report
         uses: actions/upload-artifact@v3
         with:
-          name: changes
-          path: scripts/tpip.md
+          name: tpip-report
+          path: scripts/${{ env.report_name }}
 
       - name: Print TPIP Report
-        run: cat tpip.md >> $GITHUB_STEP_SUMMARY
+        run: cat ${{ env.report_name }} >> $GITHUB_STEP_SUMMARY
         working-directory: scripts
 
       - name: Check Licenses
         run: go-licenses check . --ignore github.com/Open-CMSIS-Pack/cbuild --disallowed_types=forbidden,restricted
         working-directory: ./cmd/cbuild
+
+  commit-changes:
+    needs: [ check-licenses ]
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: Restore Changes
+        uses: actions/download-artifact@v3
+        with:
+          name: tpip-report
+
+      - name: Commit Changes
+        shell: bash
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git add .
+          git commit -m "Update documentation for ${MSG} ${GITHUB_REF_NAME}"
+          git push
+      # - name: Commit Changes

```

Comment:
Do we still need these commands being commented out?

Reply:
[Addressed review comments](https://github.com/Open-CMSIS-Pack/cbuild/pull/86/commits/57f7beb7d50ea42ea675a215fb306b49fbf6dc12)